docs/_sources/develop/build.rst.txt

```diff
@@ -0,0 +1,43 @@
+Building Extensions
```

Comment:
Do we need to clarify what an "extension" is? Perhaps it would be helpful to retitle "Extending Linux For Health" or "Adding New Capabilities to Linux for Health" and then defining what an extension or capability is in the Overview section below. For further clarity we can define an "extension" as a new Camel Route or an update to an existing route, depending on the use-case and the scope we will initially support.

This may be helpful for folks that don't read the entire documentation set.

Reply:
Good point.  I wrote those sections last and in my mind, extensions had been discussed ad infinitum.  I'll take a look.